packages/kbn-es/src/serverless_resources/security_roles.json

```diff
@@ -219,47 +331,88 @@
             "logs-*",
             "packetbeat-*",
             "winlogbeat-*",
-            ".lists*",
-            ".items*",
```

Comment:
Moved these below with `siem-signals-*` for keeping privileges for enabling detections in one group.